compilers/concrete-compiler/compiler/lib/Dialect/FHE/Analysis/MANP.cpp

```diff
@@ -845,7 +887,7 @@ static llvm::APInt getSqMANP(mlir::concretelang::FHELinalg::MatMulEintIntOp op,
       "Missing squared Minimal Arithmetic Noise Padding for encrypted operand");
 
   llvm::APInt lhsNorm = operandMANPs[0]->getValue().getMANP().value();
-  llvm::APInt accNorm = llvm::APInt{1, 1, false};
+  llvm::APInt accNorm = llvm::APInt{1, 0, false};
```

Comment:
fix: initialize the accumulator norm to 0 (it is added to some other norms later in the function)